### DISCUSSION TITLE
Mac OS 9 guest on macOS host, how to share files

### DISCUSSION THREAD

**Discussion by tehabe** [utmapp/UTM#6449] [UTM Help (macOS)]
I have an old Zip drive image which document I wanted to preserve, I could open it on Mac OS 9 and store as PDF or a newer file format with the hope to preserve it on macOS but the issue is, I can't get it out of Mac OS 9, the can only mount the ZIp image successfully as read only.

**Comment by steveparry1** [comment]
I downloaded an early version of Transmit (1.6 maybe). I saved this in a folder with other files as a disk image (or iso can't remember now). Then mounted this as a second drive in UTM preferences. Then I opened Quickftp on the host machine. Then opened Transmit on the guest in OS 9.2 and by guesswork added the correct ip addresses to Transmit and got it working. So I now use that in OS 9.2 to share files between guest and host.

**Comment by tehabe** [comment]
Yeah, that is essentially what I did in the end. I used my NAS though and as FTP client on the guest I use Anarchy. Maybe I should also try Quickftp.

**Comment by steveparry1** [comment]
Quickftp is SO easy to use and solved the problem for me.
